cmd/orchestrate/orchestrate.go

```diff
@@ -0,0 +1,296 @@
+// The orchestrate service plays the role of processing messages
+// in stand-alone operations.
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	uuid "github.com/google/uuid"
+
+	"sda-pipeline/internal/broker"
+	"sda-pipeline/internal/config"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
```suggestion

	"sda-pipeline/internal/broker"
	"sda-pipeline/internal/config"

	log "github.com/sirupsen/logrus"
	uuid "github.com/google/uuid"
```

gofmt should have caught this.

Reply:
It didn't, but I'll fix it.